src/Elastic.Serilog.Enrichers.Web/Adapters/HttpContextAccessorAdapter.cs

```diff
@@ -2,15 +2,17 @@
 // Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
 // See the LICENSE file in the project root for more information
 
-#if NETSTANDARD
-using System.Linq;
+#if NETCOREAPP
```

Comment:
I think we can use `#NET` now as the "newer" convention, not that it really matters.